anta/tests/configuration.py

```diff
@@ -57,15 +57,24 @@ async def verify_running_config_diffs(
         * result = "error" if any exception is caught
 
     """
-    device.assert_enable_password_is_not_none("verify_running_config_diffs")
-
-    response = await device.session.cli(
-        commands=[
-            {"cmd": "enable", "input": str(device.enable_password)},
-            "show running-config diffs",
-        ],
-        ofmt="text"
-    )
+    if device.enable_password is not None:
+        device.assert_enable_password_is_not_none(
```

Comment:
Remove the assertion as you have already checked line 60 that the password was not None

Reply:
Done. Thanks for the comment